src/serde/value/borrowed/de.rs

```diff
@@ -564,8 +577,8 @@ impl<'de> VariantAccess<'de> for VariantDeserializer<'de> {
     {
         match self.value {
             Some(Value::Object(v)) => visitor.visit_map(ObjectAccess {
-                i: v.iter(),
-                v: &Value::Static(StaticNode::Null),
+                i: v.into_iter(),
+                v: Value::Static(StaticNode::Null),
             }),//visit_object(v, visitor),
             // FIXME
```

Comment:
Same. Remove comments that aren't adding value.

Reply:
same :sob:

Reply:
😭